cryptoprotocol/src/main/java/org/cryptimeleon/incentivesystem/cryptoprotocol/IncentiveSystem.java

```diff
@@ -208,6 +208,7 @@ public Token handleJoinRequestResponse(IncentivePublicParameters pp, ProviderPub
      *
      * @param token             the token to update
      * @param providerPublicKey the public key of the provider
+     * @param userKeyPair the key pair of the user submitting the request, sk is the PRF input used to generate pseudorandomness
```

Comment:
sk reference confuses me

Reply:
It is there to justify why the secret key of the user is needed in join response handling.
Would you suggest rewriting that comment or removing the explanation entirely?

Reply:
I would just remove it. I rather confuses the dev who just wants to know how to use our IncentiveSystem interface :)

Reply:
I‘m in favor of separation of concerns here. If you have implementation-specific comments, put them into the body of the function, where it‘s only read by people who _want_ to see the specifics anyway :)